test/src/unit-capi-serialization-dense_array.cc

```diff
@@ -3013,120 +3013,6 @@ TEST_CASE_METHOD(
   remove_temp_dir(temp_dir);
 }
 
-TEST_CASE_METHOD(
```

Comment:
Why remove this? Is it not passing?

Reply:
The test included the following scenario:

- Open an array
- Allocate a query on that array
- Close the array
- Open the array again
- Serialize the query

But the query object internally hangs on to a pointer to the `ArraySchema` from the array it was allocated with, and that object is deleted when the array is closed the first time. This test exists in upstream also, but we should probably remove it from there as well (this is sort of a "won't fix" scenario, it violates a lot of our assumptions about object lifetimes).

Reply:
okay makes sense thanks for clarifying.